examples/enforce-gatekeeper/warning-only/README.md

```diff
@@ -27,30 +50,37 @@ $ mkdir results
 Run the function:
 
 ```shell
-$ kpt fn run --results-dir=results .
+$ kpt fn render --results-dir=results
 ```
 
 ### Expected result
 
-You won't any failure. But if you look at the structured output, you can find a
-warning about the constraint violation.
+You won't see any failure from the console. But if you look at the structured
```

Comment:
@droot why?

Reply:
Last update was 5+ days ago. The behavior may have changed already.
I will verify if this is still the latest behavior.